lib/pka.c

```diff
@@ -940,14 +973,7 @@ static pka_status_t pka_submit_cmd(pka_handle_t    handle,
         return FAILURE;
     }
 
-    if (NULL == job) 
-    {
-        // SYNC mode
-	int ret = pka_rslt_dequeue(local_info, &errors, &count);
-        if (ret)
-            PKA_DEBUG(PKA_USER, "failed to dequeue %d results\n", ret);
-    } 
-    else

```

Comment:
Are we not dequeuing the results in SYNC mode ? or is this done somewhere else ?

Reply:
There are multiple calls to pka_rslt_dequeue() in SYNC mode per cmd submit already. This is not needed.